lock-api/src/main/java/com/palantir/lock/LockRequest.java

```diff
@@ -48,8 +50,21 @@
 @Immutable public final class LockRequest implements Serializable {
     private static final long serialVersionUID = 0xf6c12b970b44af68L;
 
+    private static final AtomicReference<TimeDuration> DEFAULT_LOCK_TIMEOUT = new AtomicReference<>(
```

Comment:
DEFAULT_LOCK_TIMEOUT -> LOCK_TIMEOUT, its not default anymore since we can set it from the config.